src/BotConfig.ts

```diff
@@ -68,8 +68,7 @@ export default class BotConfig {
 		if ( !settings.request_channels ) throw 'Request channels are not set';
 		this.requestChannels = settings.request_channels;
 
-		//no check, beacause it's a boolean.
-		this.ticketUrlsCauseEmbed = settings.ticketUrlsCauseEmbed;
+		this.ticketUrlsCauseEmbed = !!settings.ticketUrlsCauseEmbed;
```

Comment:
Was this done by mistake or does the double negation do some fucky stuff in js / ts?

Reply:
Convert to boolean

Reply:
Shouldn't TypeScript be smart enough for that? `this.ticketUrlsCauseEmbed` is declared as boolean.

Reply:
Just making sure that `this.ticketUrlsCauseEmbed` is a boolean and not something like a string or `undefined`. That wouldn't really have been too much of an effect though, at least in the way it's currently used.

The rest of the config fields is not validated either though, might do that while transitioning everything towards server-specific configs.

Reply:
And we're loading the data directly from JSON here, so `settings.ticketUrlsCauseEmbed` is of type `any`. Probably wouldn't have caused too much trouble if I've left it though. Just making sure that the bot doesn't crash if the value is undefined or otherwise faulty.